src/object/obj_tx.c

```diff
@@ -2824,7 +2822,7 @@ dc_tx_per_akey_existence_task(enum obj_rpc_opc opc, daos_handle_t oh, struct dc_
 		if (unlikely(d_list_empty(&task_list))) {
 			struct dc_object	*obj;
 
-			obj_hdl2ptr(oh);
+			obj = obj_hdl2ptr(oh);
```

Comment:
is this never tested in CI? sounds weird that this was not caught before.

Reply:
Yes, this case is for that the caller claims that it will have per-akey based conditional updates, but no iods have the condition flags. Currently, we do not have related test cases in CI.